language-server-commons/src/main/java/com/palantir/ls/util/LoggerMessageTracer.java

```diff
@@ -1,72 +0,0 @@
-/*
- * Copyright 2016 Palantir Technologies, Inc. All rights reserved.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-package com.palantir.ls.util;
-
-import io.typefox.lsapi.Message;
-import io.typefox.lsapi.NotificationMessage;
-import io.typefox.lsapi.RequestMessage;
-import io.typefox.lsapi.ResponseMessage;
-import io.typefox.lsapi.services.transport.trace.MessageTracer;
-import java.io.PrintWriter;
-import java.io.StringWriter;
-import org.slf4j.Logger;
-
-public class LoggerMessageTracer implements MessageTracer {
```

Comment:
I think we still want to implement this. Right now all the requests/responses will be logged in language server logs. But right now it only logs errors https://github.com/eclipse/lsp4j/blob/bd1a4ddf97ea3a0ee9031ade9b10410bfb093999/org.eclipse.lsp4j.jsonrpc/src/main/java/org/eclipse/lsp4j/jsonrpc/json/StreamMessageProducer.java#L109

We should implement streamMessageProducer and streamMessageConsumer again
https://github.com/eclipse/lsp4j/issues/23